workspace/meterpreter.sln

```diff
@@ -33,6 +33,8 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ext_server_extapi", "ext_se
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ext_server_kiwi", "ext_server_kiwi\ext_server_kiwi.vcxproj", "{1C307A8B-A88E-43EE-8E80-01E6EFE38697}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ext_server_pageantjacker", "ext_server_pageantjacker\ext_server_pageantjacker.vcxproj", "{48092613-4C22-43D4-8965-24C8EF238720}"

```

Comment:
This answers my question about which extension did you copy from :)

Please create a new GUID for this project.